airflow/providers/amazon/aws/example_dags/example_dynamodb_sensor.py

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Example DAGs are being dropped and have all been moved into system tests [here](https://github.com/apache/airflow/tree/main/tests/system/providers/amazon/aws).  Please either update the DynamoDB system test with this new sensor if possible or create a new system test there instead of this Example DAG.

Reply:
Removed `example_dynamodb_sensor.py` from this PR for now.

Reply:
Agreed with Dennis here, it would be nice to have a system test for this, but also we have no operators for creating a Dynamo DB table or even writing to it. So it would be a lot of custom code to just test the sensor. I don't feel too strongly either way, WDYT @ferruzzi?

Reply:
I actually noticed that after I made the comment and took a peek where we could squeeze it into the existing test and... we don't have one.  Sorry, I got distracted and forgot to come back and sort that out.

I'd say it can be added into the existing [Dynamo-to-S3 system test](https://github.com/apache/airflow/blob/main/tests/system/providers/amazon/aws/example_dynamodb_to_s3.py).  Drop the new sensor right after the table_setup perhaps?

Reply:
@ferruzzi Will do.